config.example.php

```diff
@@ -7,7 +7,7 @@
 
 define('PRINTFRIENDLY_API_KEY', '');
 define('PRINTFRIENDLY_CSS_URL', get_stylesheet_directory_uri() . '/css/printfriendly_pdf.css');
-define('AC_API_URL', '');
-define('AC_API_KEY', '');
-define('AC_LIST_ID', 5);
+define('EO_API_URL', '');
+define('EO_API_KEY', '');
+define('EO_LIST_ID', 5);
 define('INSERT_POINTS', ['<h2', '<div class="fluen-after-content"']);
```

Comment:
@husenmemon , I think you did a restyle for the blog recently, is that `fluen-after-content` still a valid class to insert this widget?